pennylane/io.py

```diff
@@ -141,17 +293,71 @@ def from_qiskit(quantum_circuit, measurements=None):
             qc.cx(0, 1)
             qc.measure_all()
 
-            measurements = [qml.expval(qml.Z(0)), qml.vn_entropy([1])]
-            quantum_circuit = qml.from_qiskit(qc, measurements=measurements)
+        Then we can create a PennyLane circuit that uses this as a sub-circuit, and performs
+        additional operations conditional on the results. We can also calculate standard mid-circuit
+        measurement statistics, like expectation value, on the returned measurements:
+
+        .. code-block:: python
 
             @qml.qnode(qml.device("default.qubit"))
-            def circuit_loaded_qiskit_circuit():
-                return quantum_circuit()
+            def circuit():
+                # apply the QuantumCircuit and retrieve the measurements
+                mid_measure0, m0, m1 = qml.from_qiskit(qc)()
+
+                # conditionally apply an additional operation based on the results
+                qml.cond(mid_measure0==0, qml.RX)(np.pi/2, 0)
+
+                # return the expectation value of one of the mid-circuit measurements, and a terminal measurement
+                return qml.expval(mid_measure0), qml.expval(m1)
+
+        >>> circuit()
+        (tensor(0.5, requires_grad=True), tensor(0.5, requires_grad=True))
+
+        Furthermore, the Qiskit `IfElseOp <https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.IfElseOp>`__,
```

Comment:
```suggestion
        .. note::

            The order of mid-circuit measurements returned by `qml.from_qiskit()` in the example
            above is determined by the order in which measurements appear in the input Qiskit
            ``QuantumCircuit``.

        Furthermore, the Qiskit `IfElseOp <https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.IfElseOp>`__,
```
We haven't said this already, right?

Reply:
Ah, good catch! 🎣